test/crowdsale/CapStagedCrowdsale.test.js

```diff
@@ -0,0 +1,69 @@
+const CapStagedCrowdsale = artifacts.require('CapStagedCrowdsale');
+const BigNumber = web3.BigNumber;
+const CapStagedCrowdsaleToken = artifacts.require('CapStagedCrowdsaleToken');
+require('chai').use(require('chai-as-promised')).use(require('chai-bignumber')(BigNumber)).should();
```

Comment:
please use the style from the rest of the tests where this line is broken into multiple lines

Reply:
I've corrected that.